src/additional/postalcodeDK.js

```diff
@@ -0,0 +1,12 @@
+/* Matches Danish postcode */
+$.validator.addMethod( "postalcodeDK", function( value, element ) {
+	return this.optional( element ) || /^[0-9]{4}$/.test( value );
+}, "Please specify a valid postal code" );
+
+/*
+ * This accept all danish(DK) zip codes (1000 to 9999)
```

Comment:
Please move this comment to the top

Reply:
Oh! why did I not see that? a bigger question is why GitHub didn't notify me about your feedback